oviewer/oviewer.go

```diff
@@ -701,6 +707,48 @@ func (root *Root) toggleMouse() {
 	}
 }
 
+func (root *Root) edit() {
+	editor := root.identifyEditor()
+	editCmd := exec.Command(editor, root.Doc.FileName)
+	editCmd.Stdin = os.Stdin
+	editCmd.Stdout = os.Stdout
+	if err := editCmd.Run(); err != nil {
+		root.setMessage(ErrFailedCommand.Error())
+	}
+	//TODO: Reread the file
+	root.Doc.ReadFile(root.Doc.FileName)
+	root.viewSync()
+	root.Screen.Sync()
```

Comment:
There are currently two issues here:

1. Rereading the modified file doesn't work.
2. "Some" keypresses in the opened editor are hijacked by ov (and therefore swallowed) and the last keypress in the opened editor will always be interpreted by ov (e.g. quitting vi via `:wq` will quit ov, since it interprets the `q`). We would actually need to suspend interpreting keypresses before opening the editor and resume it again afterwards. But I don't know how to do this with tcell/cbind.

Help with both problem is very appreciated.